curlylint/rules/django_block_translate_trimmed/django_block_translate_trimmed_test.json

```
@@ -0,0 +1,90 @@
+[
+  {
+    "label": "Using blocktranslate with trimmed",
+    "template": "{% blocktranslate trimmed %} some value {% endblocktranslate %}",
+    "example": true,
+    "config": true,
+    "output": []
+  },
+  {
+    "label": "Using blocktranslate without trimmed",
+    "template": "{% blocktranslate %} some value {% endblocktranslate %}",
+    "example": true,
+    "config": true,
+    "output": [
+      {
+        "file": "test.html",
+        "column": 1,
+        "line": 1,
+        "code": "django_block_translate_trimmed",
+        "message": "`{% block translate %}` must use the `trimmed` option"
```

question: should this not be 
```suggestion
        "message": "`{% blocktranslate %}` must use the `trimmed` option" 
```
?
Ditto for all other messages

---

Yeah. Not sure what to do here because the parser spits out the tag with the word `block` as a separate item. 

Maybe I can try to put the full node back in the message instead.

---

fixed it I think, thanks for the push @zerolab